### PR TITLE
Docs for `newEventWithTrigger`. Addresses #3

### PR DESCRIPTION
**Pull request by schell**
Just some docs I applied from stuff you've said in irc, @ryantrinkle .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Reflex
+======
+Practical Functional Reactive Programming
+
+Resources
+---------
+[try reflex](https://github.com/ryantrinkle/try-reflex)
+
+[reddit/s/reflexfrp](http://www.reddit.com/r/reflexfrp)
```
Comment (ryantrinkle) [inline README.md]: Is this meant to be reddit/s/reflexfrp or reddit/r/reflexfrp?

Comment (schell) [inline README.md]: It's meant to be 'r'. I typo'd.

**Comment by cocreature** [comment]
In case @ryantrinkle misses this otherwise, the typo got fixed. (I'll never understand why github doesn't send notifications for new commits on pull requests)

**Comment by schell** [comment]
Ah, yes - I assumed it would. Thanks :)

**Comment by ryantrinkle** [comment]
Merged to develop; thanks!

**Pull request closed without merging**